src/lib/util/rbtree_tests.c

```diff
@@ -78,6 +87,42 @@ static void test_rbtree_iter(void)
 	pthread_mutex_unlock(&t->mutex);
 
 	talloc_free(t);
+
+	/*
+	 *	Build a tree for pre- and post-order iterator tests
+	 */
+	t = rbtree_alloc(NULL, fr_rb_test_node_t, node, fr_rb_test_cmp, NULL, RBTREE_FLAG_LOCK);
```

Comment:
Could you break pre and post order tests out into separate functions and add them to the test array at the bottom (maybe rename the existing test too). Other than that everything looks good!

Reply:
Will do, thanks!

Reply:
Changes made and pushed.